webapp/tests/Controller/Jury/ContestControllerTest.php

```diff
@@ -0,0 +1,204 @@
+<?php declare(strict_types=1);
+
+namespace App\Tests\Controller\Jury;
+
+use App\Tests\BaseTest;
+
+class ContestControllerTest extends BaseTest
+{
+    protected static $roles       = ['admin'];
+    protected static $baseUrl     = '/jury/contests';
+    protected static $baseUrlLong = 'http://localhost/jury/contests';
+    protected static $addButton   = ' Add new contest';
+
+    /**
+     * Test that jury contest overview page exists
+     * @var string role The role of the user
+     * @var int statuscode The statuscode the current user should get
+     * @var string baseContest The basecontest we use throughout these tests
+     * @dataProvider provideBasePage
+     */
+    public function testContestsOverview (
+	string $role,
+	int $statuscode,
+	string $baseContest) : void
+    {
+    	static::$roles = [$role];
+        $this->logIn();
```

Comment:
The new tests log in automatically, so i think if you want something like this best is to first log out

Reply:
they login as admin I assume?

Reply:
As what you define as the static $roles variable